DXRCore/DeusEx/Classes/DXRMenuSetupRando.uc

```diff
@@ -245,21 +245,24 @@ function BindControls(optional string action)
     NewMenuItem("Enemy Weapons Variety %", "Should enemies be using weapons that normally exist in the map?");
     Slider(f.moresettings.enemies_weapons, 0, 100);
 
+    NewMenuItem("Robot Weapons Rando %", "Allow robots to get randomized weapons.");
+    Slider(f.settings.bot_weapons, 0, 100);
+
     NewMenuItem("Non-Human Chance %", "Reduce the chance of new enemies being non-humans.");
     Slider(f.settings.enemies_nonhumans, 0, 100);
 
     NewMenuItem("Enemy Respawn Seconds", "How many seconds for enemies to respawn. Leave blank or 0 to disable.");
     Slider(f.settings.enemyrespawn, 0, 10000);
 
+    NewMenuItem("Reanimation Seconds", "Approximately how many seconds for corpses to rise from the dead as zombies. Leave blank or 0 to disable.");
+    Slider(f.moresettings.reanimation, 0, 10000);
```

Comment:
10,000 seconds is a bit high for a maximum? lol

Reply:
I don't see why the maximum should be different from respawning, even though the normal value is a lot lower. 167 minutes might be too high for both though?

Reply:
yea, I think both could be lowered to 3600 seconds or something, an identifiable number that people could understand?